src/main/java/appeng/facade/FacadePart.java

```diff
@@ -461,7 +461,7 @@ public boolean isTransparent()
 		return !blk.isOpaqueCube();
 	}
 
-	@Nullable
+	private @Nullable
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)